src/domains/queries.ml

```diff
@@ -128,6 +128,9 @@ type _ t =
   | MayBeTainted: AD.t t
   | MayBeModifiedSinceSetjmp: JmpBufDomain.BufferEntry.t -> VS.t t
   | TmpSpecial:  Mval.Exp.t -> ML.t t
+  | MayBeOutOfBounds: varinfo * int * exp * binop -> ID.t t
+  | MayOverflow : exp -> MustBool.t t
```

Comment:
I didn't think of that because there is only one analysis that answers this query. But you are correct, that ```MustBool``` is not that fitting. I have changed this to ```MayBool``` in d2866ce9493571ba0074c4171febcbde2b44efca